### PR TITLE
Add scroll to top after dismissing accident x-sell

### PR DESCRIPTION
**Pull request by gustaveen**
## Describe your changes
- Adjust spacing on `CartPage`
- Add scroll to top after dismissing x-sell accident

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by gustaveen** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #3779** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3779?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3779?utm_source=stack-comment).

File: apps/store/src/components/QuickAdd/DismissButton.tsx [hunk 1]
```diff
@@ -10,6 +10,7 @@ export const DismissButton = () => {
   const handleDismiss = () => {
     datadogRum.addAction('Quick Add Hide')
     setShow(false)
+    window.scrollTo({ top: 0, behavior: 'smooth' })
```
Comment (guilhermespopolin) [inline apps/store/src/components/QuickAdd/DismissButton.tsx:13]:
```suggestion
    window.scrollTo({ top: 0 })
```
We already set scroll behaviour globally in [`globalStyles`](https://github.com/HedvigInsurance/racoon/blob/main/packages/ui/src/lib/globalStyles.ts#L31-L39)

image

and we're taking user's settings into account for that as well. I think if we pass `behavior` to `window.scrollTo` we'd be overriding that, which would make users that opt in for _reduced motion_ still experience some motion here.

**Review by guilhermespopolin — APPROVED** [review]
Well done. Just left a comment.

File: apps/store/src/components/QuickAdd/DismissButton.tsx (hunk 1 above)
Comment (gustaveen) [reply apps/store/src/components/QuickAdd/DismissButton.tsx:13]: Ah didn't know that, I'll remove it 🙌

**Comment by gustaveen** [comment]
### Merge activity

* **Dec 21, 8:50 AM**: @@gustaveen merged this pull request with [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3779).

**Review by alebedev — APPROVED** [review]

**Pull request merged**